event-controller-template.yml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
 kind: Template
 labels:
-  template: event-watcher
+  template: event-controller
```

Comment:
@oybed to me, once you're in openshift, using `openshift` in the name of things becomes redundant.